src/main/java/Duke.java

```diff
@@ -1,10 +1,196 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+import javafx.application.Application;
```

Comment:
I like how your import statements are grouped according to their functionality which makes it easier to read!